docs/experiments.md

```diff
@@ -0,0 +1,21 @@
+## Experiments
+
+`Experiments` is an API to change the behaviour of the library [on the fly](https://martinfowler.com/articles/feature-toggles.html).
```

Comment:
Experiments look exactly as an approach to[hypothesis-driven-development](https://www.thoughtworks.com/insights/articles/how-implement-hypothesis-driven-development)
and FF are the RealTime solution to manipulate the hypothesis. 

Hypothesis still maybe controlled manually / automatically (if some exceptions-> rollback(aka. circuit breaker for new hypothesis)